presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/DockerContainer.java

```diff
@@ -252,7 +262,9 @@ private void copyFileFromContainer(String filename, Path rootHostPath)
         try (DockerClient client = DockerClientFactory.lazyClient()) {
             InvocationBuilder.AsyncResultCallback<Statistics> callback = new InvocationBuilder.AsyncResultCallback<>();
             client.statsCmd(getContainerId()).exec(callback);
-            return Optional.ofNullable(callback.awaitResult());
+
+            return Optional.ofNullable(executor.get(callback::awaitResult))
+                    .map(Statistics.class::cast);
```

Comment:
It is not obvious to me why we need cast here. Someone stupid. Either me or java compiler.

Reply:
executor is FailsafeExecutor<Object> (thus .get() returns Object

Reply:
Not really:
![image](https://user-images.githubusercontent.com/890111/94015298-f7187980-fdac-11ea-96d5-9c59781a74e0.png)